src/sentry/templates/sentry/twofactor.html

```diff
@@ -54,3 +55,19 @@ <h3>Two-Factor Authentication</h3>
     </div>
   </div>
 {% endblock %}
+
+{% block scripts_bottom %}
+    {{ block.super }}
+    {% script %}
+    <script>
+    document.addEventListener('DOMContentLoaded', function() {
+        var input = document.querySelector('input[name="otp"]');
+        input.addEventListener('change', function() {
+            if (input.value.length >= 6) {
+                input.form.submit();
+            }
+        });
+    });
```

Comment:
I think you can write es5 here now
```suggestion
    document.addEventListener('DOMContentLoaded', function() {
        const input = document.querySelector('input[name="otp"]');
        input.addEventListener('change', function() {
            if (input.value.length >= 6) {
                input.form.submit();
            }
        });
    });
```